Sources/BadgeGenerator/BadgeGenerator.swift

```diff
@@ -121,7 +121,10 @@ struct BadgeView: UIViewRepresentable {
     private let view = UIView()
     
     init(view: (UIView) -> ()) {
-        view(self.view)
+    private let previewView = UIVIew()
+    
+    init(view: UIView) -> ()) {
+        view(previewView)
     }
```

Comment:
```suggestion
    private let previewView = UIVIew()

    init(view: UIView) -> ()) {
        view(previewView)
    }
Commenting on lines 123 to 128
```

Reply:
@codebendr-product-designer - it looks like my suggestion was malformed, this is causing build errors

Reply:
Yes, I just noticed that too...I can quickly fix them.

Reply:
thanks, I'm unable to commit directly to your fork because I haven't set the repo up to be able to do that just yet